OpenRA.Mods.Common/Activities/HarvestResource.cs

```diff
@@ -32,6 +33,7 @@ public class HarvestResource : Activity
 		public HarvestResource(Actor self, CPos targetCell)
 		{
 			harv = self.Trait<Harvester>();
+			aircraft = self.TraitOrDefault<Aircraft>();
```

Comment:
```suggestion
			aircraft =move as Aircraft;
```
(And need to re-order this below the definition of `move`.)